css/style.css

```diff
@@ -0,0 +1,435 @@
+/*! normalize.css v4.1.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Sorry.. I gitignored from a release perspective, and not a development perspective. Doi! FIXED.